### ISSUE TITLE
Reduce Top Drawer Opened Spacing

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#39]
# Description
```
Please provide a short description of what you have encountered below.
```
Factor in mobile's control panel bar. That pushes the entire screen down 10%.

![Screenshot_20221226_165524_com.android.chrome.jpg](https://user-images.githubusercontent.com/7259572/209528369-7c54c9ee-e100-4ec2-bc83-1a05f9a2526b.jpg)
![Screenshot_20221226_165506_com.android.chrome.jpg](https://user-images.githubusercontent.com/7259572/209528381-243c33af-1754-48dd-ad64-8e1bc35338ff.jpg)

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```
Top drawer can be properly interacted even with control panel disruption.

# Current Behavior
```
Please specify the current behavior of software below.
```

Top drawer can't be closed due to control panel's disruption and pwa install prompt.

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Fixed in 85adf689fefe3430024a552866e2daa02211ae0f

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
